### PR TITLE
docs: Update arc instructions, add troubleshooting

### PR DESCRIPTION
**Pull request by ejlee3**
Closes: #366

## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [x] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [x] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## Issue this PR will close

close: #366, #367

## Test Instructions if applicable
run `make serve-docs`

check the hardware page, troubleshooting page

### CODE REVIEW

**Review by brian-intel — APPROVED** [review]

File: docs_src/hardwaresetup.md [hunk 1]
```diff
@@ -27,17 +28,6 @@ Click the links for corresponding set up instructions.
...
-    2. Install [Docker Engine](https://docs.docker.com/engine/install/ubuntu/)
-
-    3. [**Optional**] Install [Docker Compose v2](https://docs.docker.com/compose/), if using the docker compose feature
-    
-    4. Install [Git](https://git-scm.com/download/linux)
-
-    3. [Set up the pipeline](./OVMS/pipelinesetup.md)
-
 === "Intel® Data Center GPU Flex 140/170"
     1. Download [Ubuntu 22.04](https://releases.ubuntu.com/22.04/) and follow these [installation steps](https://ubuntu.com/tutorials/install-ubuntu-desktop#1-overview).
```
Comment (jim-wang-intel) [inline docs_src/hardwaresetup.md:32]: have we figured out the deprecated html page  issue?

Comment (jim-wang-intel) [reply docs_src/hardwaresetup.md:32]: this https://dgpu-docs.intel.com/installation-guides/ubuntu/ubuntu-jammy-dc.html page is deprecated.

Comment (brian-intel) [reply docs_src/hardwaresetup.md:32]: You can point to https://dgpu-docs.intel.com/driver/installation.html#ubuntu-install-steps

File: docs_src/troubleshooting.md [hunk 2]
```diff
@@ -13,3 +13,12 @@ A: The performance of pipeline benchmarking strongly depends on the models.  Spe
...
     }
 ```
+
+Q: What happens if the system keeps crashing when building the `dlstreamer-realsense` image?
+
+A: Some systems may run into issues with memory when building the `dlstreamer-realsense` image. In the `Dockerfile.dlstreamer` file, change the make command to not use the `-j` threading option.
+
+```diff
+- make -j"$(cat < /proc/cpuinfo |grep -c proc)" &&
++ make "$(cat < /proc/cpuinfo |grep -c proc)" &&
```
Comment (jim-wang-intel) [inline docs_src/troubleshooting.md:23]:
```suggestion
+ make &&
```

File: docs_src/hardwaresetup.md [hunk 3]
```diff
@@ -27,19 +28,8 @@ Click the links for corresponding set up instructions.
...
-    
-    4. Install [Git](https://git-scm.com/download/linux)
-
-    3. [Set up the pipeline](./OVMS/pipelinesetup.md)
-
 === "Intel® Data Center GPU Flex 140/170"
-    1. Download [Ubuntu 22.04](https://releases.ubuntu.com/22.04/) and follow these [installation steps](https://ubuntu.com/tutorials/install-ubuntu-desktop#1-overview).
+    1. Download [Ubuntu 22.04](https://releases.ubuntu.com/22.04/) and follow these [installation steps](https://dgpu-docs.intel.com/driver/installation.html#ubuntu-install-steps).
 
     2. Update the [Kernel](https://dgpu-docs.intel.com/installation-guides/ubuntu/ubuntu-jammy-dc.html)
```
Comment (jim-wang-intel) [inline docs_src/hardwaresetup.md:34]: For this step, let's either remove this or point to the installation steps above, [this page https://dgpu-docs.intel.com/installation-guides/ubuntu/ubuntu-jammy-dc.html](https://dgpu-docs.intel.com/installation-guides/ubuntu/ubuntu-jammy-dc.html) is deprecated

Comment (ejlee3) [reply docs_src/hardwaresetup.md:34]: ok. that page is deprecated too - the kernel install is for a really old kernel version 5.15 there... latest ubuntu 22.04 would install a 6.X kernel

**Review by brian-intel — APPROVED** [review]

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Pull request merged**